pkg/plugins/policies/core/matchers/dataplane_test.go

```diff
@@ -206,4 +206,21 @@ var _ = Describe("MatchedPolicies", func() {
 		},
 		generateTableEntries(filepath.Join("testdata", "matchedpolicies", "meshgateways")),
 	)
+
+	DescribeTable("should match MeshGateways for dpp without meta",
+		func(given testCase) {
+			dpp := readDPP(given.dppFile)
+			dpp.SetMeta(nil)
+
+			resources, _ := readPolicies(given.policiesFile)
+
+			policies, err := matchers.MatchedPolicies(meshaccesslog_api.MeshAccessLogType, dpp, resources)
+			Expect(err).ToNot(HaveOccurred())
+
+			bytes, err := yaml.Marshal(policies.GatewayRules)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(bytes).To(test_matchers.MatchGoldenYAML(given.goldenFile))
+		},
+		generateTableEntries(filepath.Join("testdata", "matchedpolicies", "dpp_without_meta")),
+	)
```

Comment:
I think you don't need to add this new test case. The point of the generateTableEntries is that you can simply add tests cases by sending: input, policies --> golden.

Reply:
yes, but I needed to clean dpp meta, and you cannot create dpp without meta from file